extension/scripts/features/itemredirect/ttBazaarRedirect.js

```diff
@@ -0,0 +1,59 @@
+"use strict";
+
+(async () => {
+	featureManager.registerFeature(
+		"Bazaar Redirect",
+		"bazaar",
+		() => settings.pages.bazaar.redirects,
+		null,
+		addHighlight,
+		removeHighlight,
+		{
+			storage: ["settings.pages.bazaar.redirects"],
+		},
+		null
+	);
+
+	async function addHighlight() {
+		await requireElement("[class*='rowItems_'] [class*='item_']");
+		const params = getSearchParameters();
+		if (params.has("tt_itemid") && !(await checkMobile())) {
+			const itemID = params.get("tt_itemid");
+			const itemPrice = params.get("tt_itemprice");
+			const itemName = torndata.items[itemID].name;
+
+			let foundItem = false;
+			for (const item of document.findAll("[class*='rowItems_'] [class*='item_']")) {
+				if (
+					item.find("[class*='name_']").innerText.trim() === itemName &&
+					item.find("[class*='price_']").lastChild.textContent.replace(/[\n\$, ]/g, "") === itemPrice
+				) {
+					foundItem = true;
+					item.classList.add("tt-flash");
+					item.scrollIntoView({ behavior: "smooth" });
+					break;
+				}
+			}
+
+			if (!foundItem) {
+				document.find(".info-msg-cont .msg").appendChild(
```

Comment:
Should it still show the message anymore, as we are searching automatically now?

Reply:
Removed message.